presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -44,6 +44,14 @@ Array Functions
 
     Remove duplicate values from the array ``x``.
 
+.. function:: array_dupes(array(bigint)) -> array(bigint)
+
+    Returns an array with elements that are duplicated in the ``array``. Each element is repeated only once and may be in any order.
+
+.. function:: array_dupes(array(varchar)) -> array(varchar)
+
+    Returns an array with elements that are duplicated in the ``array``. Each element is repeated only once and may be in any order.
+
```

Comment:
You don't need to repeat for every type. Just make it ARRAY<T> like we do for other functions.

Reply:
because that's the only 2 versions of the functions we spell out. It's probably nice to support generic types for these builtin SQL functions at least.